lib/shifts.rb

```diff
@@ -2,65 +2,49 @@
 
 class Shifts
 
-    def initialize
-      @shifts = shifts
-      @keys = keys
-      @offsets = offsets
-    end
-
     def random_number
```

Comment:
Not covered on simplecov.... not sure why or how to make it covered due to randomness ....